main.js

```diff
@@ -0,0 +1,102 @@
+var board = {
+  // Initialize starting board for game
+  startingPegBoard: [
+                      [5, 4, 3, 2, 1],
+                      [],
+                      []
+                    ],
+  
+  // Initialize game board
+  currentPegBoard: [
+                    [5, 4, 3, 2, 1],
+                    [],
+                    []
+                   ],
+
+  // Start a game with directions
+  startGame: function() {
+    console.log(`TOWERS OF HANOI`);
+    console.log(`Directions: Towers of Hanoi is an ancient puzzle played with three stacks and five differently-sized discs. At the start, all the discs are in the leftmost stack, with the largest disc on the bottom and the smallest on the top. The object is to get all the discs, stacked from biggest to smallest over to the right most peg with the following rules:`);
+    console.log(`- You can only move one disc per move, and you can only move the top disc on a stack.`);
+    console.log(`- You can't move a larger disc on top of a smaller disc. `);
+    console.log(`- Discs can only be moved to empty pegs or on top of larger discs.`); 
+    console.log(`- To move a disc, type into the console board.moveDisc(numberOne, numberTwo). numberOne is the peg you want to move from. numberTwo is disc you want to move the peg to. For example, board.moveDisc(1, 2) would move a disc from peg 1 to peg 2. Good luck!`);
+    this.printBoard(this.currentPegBoard);
+  },
+
+  // Moves discs between pegs
+  moveDisc: function (startPeg, endPeg) {
+    // Checks that startPeg and endPeg are between 1 - 3
+    if (startPeg < 1 || startPeg > 3 || endPeg < 1 || endPeg > 3) {
+      console.log('Please enter a startPeg and/or endPeg value between 1 and 3.');
+      this.printBoard(this.currentPegBoard); 
+      return;
+    }
+
+    // Define starting and ending pegs
+    var currentPeg = startPeg - 1;
+    let endingPeg = endPeg - 1;
+    var startingPegValues = this.currentPegBoard[startPeg - 1];
```

Comment:
it looks like the `currentPeg` and `endingPeg` variables defined above could be used to index the board here